### ISSUE TITLE
pmbootstrap zap -m: also delete old packages

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/pmbootstrap#1241]
Example:
* Package `hello-world` has version `1-r4`
* build that package
* bump the version to `2-r0`
* build that package again
* Run `pmbootstrap zap -m`
* It goes through the APKINDEX, which only has the `2-r0` package the way we currently parse it, and therefore it assumes that there is no outdated package.

**Comment by drebrez** [comment]
@ollieparanoid not sure but maybe `zap -m` never deleted older packages but only newer

**Comment by ollieparanoid** [comment]
Oh, you're right:
```
  -m, --mismatch-bins  also delete binary packages that are newer than the
                       corresponding package in aports
```

Then it's not a bug, but a feature request (which would be very useful for the binary repository, since the space fills up quickly when we don't delete old packages ever).

**Comment by drebrez** [comment]
@ollieparanoid is this maybe fixed by #1306 ?

**Comment by ollieparanoid** [comment]
It is indeed, thanks!
